### PR TITLE
Use SDK CFLAGS for lspci

### PR DESCRIPTION
**Pull request by nanonyme**

### CODE REVIEW

**Comment by nanonyme** [comment]
bot, build

**Comment by nanonyme** [comment]
bot, build

**Comment by nanonyme** [comment]
bot, build

**Comment by nanonyme** [comment]
bot, build

**Comment by nanonyme** [comment]
bot, build

File: com.valvesoftware.Steam.json [hunk 1]
```diff
@@ -96,6 +96,9 @@
         {
             "name": "lspci",
             "no-autogen": true,
+	    "make-args": [
```
Comment (gasinvein) [inline com.valvesoftware.Steam.json]: Tab/spaces mix here.
```suggestion
            "make-args": [
```

**Review by gasinvein — CHANGES_REQUESTED** [review]

File: com.valvesoftware.Steam.json (hunk 1 above)
Comment (nanonyme) [reply com.valvesoftware.Steam.json]: Sure, that's not the reason why this doesn't work though.

**Comment by nanonyme** [comment]
bot, build

File: com.valvesoftware.Steam.json (hunk 1 above)
Comment (gasinvein) [reply com.valvesoftware.Steam.json]: Just noticed that and commented, sorry.

Comment (nanonyme) [reply com.valvesoftware.Steam.json]: Yeah, it's a good catch. I'm using Emacs and it wants to always put tabs into JSON. Anyhow, completely rewritten, I couldn't manage to get CFLAGS to go through without buildsystem simple (lspci isn't really an autotools project anyway, it just has a Makefile)

**Pull request merged**